### ISSUE TITLE
Mekanism Copper does not process in mekanism machines

### ISSUE THREAD

**Issue by MrGrimmReaper28** [AllTheMods/ATM-5#238]
Mekanism copper does not process into ingots when using the tripling process - copper dust cannot be smelted by smelting factory

Mekanism copper should go from copper dust into ingots when smelted.

## Current Behavior
Cannot place mekanism copper dust into mekanism Smelting factories

## Steps to Reproduce

1. Produce copper dust - NOT dirty copper dust
2. Attempt to place into Smelting factories from mekanism

**Comment by NightzAngel** [comment]
Yeah I noticed that as well, it doesn't even cook in a vanilla furnace
